data/remnant/remnant jobs.txt

```diff
@@ -609,10 +609,9 @@ mission "Remnant: Rescue 4"
 	npc accompany save
 		system
 			attributes "graveyard"
-			not attributes "inaccessible"
 			not government "Remnant"
-			not system "Patir"
-			not system "Aki'il"
+			near "Nona" 100
```

Comment:
```suggestion
			near "Nona" 20
```

Is there any particular reason to use such a massive value? It would probably be better to go with a more reasonable value.

Reply:
No particular reason.